src/test/java/uk/gov/hmcts/reform/cpo/service/mapper/CasePaymentOrderMapperTest.java

```diff
@@ -136,6 +136,7 @@ void successfulMergeUpdateRequestIntoEntityMapping() {
         assertNotNull("Merged created timestamp should be set", entity.getCreatedTimestamp());
         assertNotEquals("Merged created timestamp should have changed",
                         originalCreatedTimestamp, entity.getCreatedTimestamp());
+
```

Comment:
Remove 'CreatedTimestamp' checks from both merge checks if we hve now remove the change.

Reply:
We should also adjust Service unit test for Create and Update to verify:
* the correct value is set for history_Exists
* the CreatedTimestamp is correctly updated prior to call to save for update.